### PR TITLE
edited for library carpentry ZA

### PR DESCRIPTION
**Pull request by Kakaalikins**
Please accept me :)

### CODE REVIEW

**Comment by cmacdonell** [comment]
Why do you think this is necessary?

File: _config.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 carpentry: "swc"
 
 # Overall title for pages.
-title: "Lesson Title"
+title: "Library Carpentry Lesson for Data Preservation"
```
Comment (cmacdonell) [inline _config.yml:9]: You misspelled carpentry

Comment (Kakaalikins) [reply _config.yml:9]: :'(

**Comment by johannvanwyk** [comment]
Actually the word was spelled correctly

**Comment by WillemijndK** [comment]
The title says more about the content

**Pull request merged**